website/sidebars.json

```diff
@@ -1,6 +1,6 @@
 {
   "docs": {
-    "Guides": ["introduction", "installation", "new-project", "migrating", "daily-dep"],
+    "Guides": ["introduction", "installation", "new-project", "migrating", "daily-dep", "uninstalling"],
```

Comment:
note to @carolynvs: tada! here is the way to add a link to a new doc page on the website :)

Reply:
Yeah! 🎉